src/x86/x86_model.py

```diff
@@ -893,45 +893,65 @@ class X86NonCanonicalAddress(X86FaultModelAbstract):
     """
      Load from non-canonical addresss
     """
-    last_faulty_addr: int
+    fauty_instruction_addr: int
```

Comment:
Code style violation again. Could you run the tests on this PR? flake8 should catch all of those automatically